src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -30,18 +31,20 @@ class JsonAdaptedPerson {
     private final String phone;
     private final String email;
     private final String address;
+    private final String metadata;
```

Comment:
You sure only a string is enough, wouldn't it be better to make a JSONAdaptedMetaData?